analysers/analyser_merge_power_line_FR.py

```diff
@@ -32,24 +32,27 @@ def __init__(self, config, logger = None):
             title = T_('Power line not integrated'))
 
         self.init(
-            'https://odre.opendatasoft.com/explore/dataset/lignes-aeriennes-rte-nv/information',
```

Comment:
Please, ca you remove all this quoting change, please.

Reply:
It's not consistent with other analyzers, should I put simple quote everywhere?

Reply:
The target is simple quote every where. But not sure is still the target.
Please avoid this kind of changing along other changings.

Reply:
Restored